server.js

```diff
@@ -2,6 +2,23 @@ const express = require("express");
 const app = express();
 const cors = require("cors");
 const PORT = 4000;
+const sqlite = require("sqlite3");
+
+let db = new sqlite.Database("../shopping-list-database/database.sqlite", err => {
```

Comment:
Would be better to use the local directory not something above...
Also add database.sqlite to .gitignore

Reply:
I will remove the database repository from my GitHub then.